CHANGELOG.md

```diff
@@ -270,9 +272,10 @@
   [#10687](https://github.com/Kong/kong/pull/10687)
 - **Oauth2**: prevent an authorization code created by one plugin instance to be exchanged for an access token by a different plugin instance.
   [#10011](https://github.com/Kong/kong/pull/10011)
-- **gRPC gateway**: fixed an issue that empty arrays in JSON are incorrectly encoded as `"{}"`; they are
-now encoded as `"[]"` to comply with standard.
+- **gRPC gateway**: fixed an issue that empty arrays in JSON are incorrectly encoded as `"{}"`; they are now encoded as `"[]"` to comply with standard.
   [#10790](https://github.com/Kong/kong/pull/10790)
+- **loggly & tcp-log & udp-log**: fix a potential issue that could cause socket leaks.
```

Comment:
Note that this change was added to the wrong version in the changelog - it should have gone into #unreleased